samza-api/src/main/java/org/apache/samza/checkpoint/KafkaStateChangelogOffset.java

```diff
@@ -26,57 +26,58 @@
  * Checkpointed changelog offset has the format: [checkpointId, offset], separated by a colon.
  */
 @InterfaceStability.Unstable
-public class CheckpointedChangelogOffset {
+public class KafkaStateChangelogOffset {
```

Comment:
IIUC, this should implement StateCheckpointMarker interface?

Reply:
this will be deprecated in favor of `KafkaStateCheckpointMarker`